packages/connectivity/src/scp-cf/token-accessor.spec.ts

```diff
@@ -47,6 +49,51 @@ describe('token accessor', () => {
       expect(actual).toBe(expected);
     });
 
+    it('considers custom timeout for client credentials token', async () => {
+      const jwt = signedJwt({
+        iss: 'https://testeroni.example.com'
+      });
+      async function doDelayTest(enableCircuitBreaker: boolean) {
```

Comment:
```suggestion
      async function doDelayTest() {
```

Reply:
I think `enableCircuitBreaker` has to be passed as an option to `serviceToken` below.

Reply:
I think `enableCircuitBreaker` has to be passed as an option in the call to `serviceToken`.